src/v/cluster/controller_log_limiter.cc

```diff
@@ -51,6 +56,35 @@ void group_limiter::update_capacity() {
     }
 }
 
+void group_limiter::setup_public_metrics() {
+    if (config::shard_local_cfg().disable_public_metrics()) {
+        return;
+    }
+
+    namespace sm = ss::metrics;
+    _public_metrics.add_group(
+      "cluster:controller_limit",
+      {sm::make_gauge(
+         ssx::sformat("{}_requests_dropped", _group_name),
+         [this] {
+             auto dropped = _dropped_requests_amount;
+             _dropped_requests_amount = 0;
```

Comment:
use `ss::make_rate` and  return `_dropped_requests_amount` value, derivation will be done in prometheus.

Reply:
changed to `make_counter`

Reply:
This hasn't changed yet